mandatory/1-syntax-errors.js

```diff
@@ -1,18 +1,15 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
-  return a + b + c;
+function addNumbers(a, b, c) {
+return a + b + c;
+}
+function introduceMe(name, age) {
+  return "Hello, my name is " + '${name}' + "and I am " + '${age}' + " years old";
 }
-
-function introduceMe(name, age)
-  return "Hello, my name is " + name "and I am " age + "years old";
-
 function getTotal(a, b) {
-  total = a ++ b;
-
-  return "The total is total";
+  total = a + b;
+  return "The total is ${total}";
```

Comment:
Close - you're using template string syntax here, but to make it work you need to use backticks ` instead of " to surround the string.

Reply:
?

Reply:
If you don't use backticks then `${total}` won't be replaced by the value of total. For instance:
```
let a = 34;
console.log("a is ${a}");   // will print: a is ${a}
console.log(`a is ${a}`);   // will print: a is 34
```